libs/community/langchain_community/vectorstores/pathway.py

```diff
@@ -0,0 +1,187 @@
+"""
+Pathway vector store server and client.
+
+
+The PathwayVectorServer builds a pipeline which indexes all files in a given folder,
+embeds them and builds a vector index. The pipeline reacts to changes in source files,
+automatically updating appropriate index entries.
+
+The PathwayVectorClient implements the LangChain VectorStore interface and queries the
+PathwayVectorServer to retrieve up-to-date documents.
+
+"""
+
+from typing import Callable, List, Optional, Tuple
+
+from langchain_core.documents import BaseDocumentTransformer, Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VectorStore
+
+IMPORT_PATHWAY_ERROR_MESSAGE = (
+    "Could not import pathway python package. "
+    "Please install it with `pip install pathway`."
+)
+
+
+class PathwayVectorServer:
```

Comment:
Let's remove this and instead document how to run it separately in the docs

Reply:
@efriis where would you propose to move it?

Unlike a classical vector database, the pathway vector store server is a single process that runs a full document ingestion pipeline and index accessible using a REST API, so it replaces a database + processing pilpeline + orchestrator. The `PathwayVectorServer` class in `libs/community/langchain_community/vectorstores/pathway.py` is a wrapper that allows easy usage of Langchain's document processing classes. I think it is beneficial - LangChain users get an easy way to build always up-to-date chatbots without having to learn another set of document processing tools.

Reply:
Hey there! Let's just have the client in community, and you can document how to run the backend server in the docs + the template readme?